js/README.md

```diff
@@ -74,3 +74,19 @@ Build and test:
 yarn run build
 yarn run test
 ```
+
+## Release Notes
+
+## Version 1.5.0
+This release contains new features, updates to code samples and new configuration options.
+
+### New features
+- Add option to specify the launch mode.
```

Comment:
should there be a one line explanation for each feature on scenario and problem it solves or advantage it provides?

Reply:
those are the same release notes from internal SDK